.github/workflows/release.yml

```diff
@@ -0,0 +1,33 @@
+name: Release
+on:
+  push:
+    branches:
+      - main
+  pull_request: {}
+  release:
+    types:
+      - published
+jobs:
```

Comment:
I added https://github.com/InfinityFlowApp/keycloak-event-listener-temporal/issues/4 to setup dependency caching to speed up builds.